### PR TITLE
Update governance.md

### PR DESCRIPTION
**Pull request by afrittoli**
Tekton goverance is not in bootstrap mode anymore.
Update the document to reflect that and add references existing
documents produced by the governance committee.

The election part is mostly taken from https://docs.google.com/document/d/1LK7V6MVzp07CMW8uVxpmvCvnN9vh2W16PAPCQMunvbU/edit#

### CODE REVIEW

File: governance.md
Comment (dlorenc) [inline governance.md:51]: nit: typo, missing space

Comment (afrittoli) [reply governance.md:51]: Nice catch, thanks

File: governance.md
Comment (bobcatfish) [inline governance.md:17]: andrew was elected in the feb 2020 election as well

File: governance.md
Comment (bobcatfish) [inline governance.md:24]: does it makes sense to mention the target date for the next election? i think it'll be me and @vdemeester and im guessing ~ feb 2021? (yikes!) np if we just want to leave that implied

File: governance.md
Comment (bobcatfish) [inline governance.md:73]: are we also responsible for processes like TEP? (not sure)

File: governance.md
Comment (bobcatfish) [inline governance.md:91]: I think thanks to https://github.com/cncf/devstats/issues/237 we can now see all contributors in this view (or at least it isnt limited to 20 now)

File: governance.md
Comment (bobcatfish) [inline governance.md:92]: i guess we could, i think k8s uses 20, 15 doesnt seem too far off

File: governance.md
Comment (bobcatfish) [inline governance.md:98]: is it worth including the nomination process and the email address here as well? otherwise we'll have to dig it up again next time

File: governance.md
Comment (bobcatfish) [inline governance.md:121]: oh interesting, i totally missed this part before. makes sense.

File: governance.md
Comment (bobcatfish) [inline governance.md:118]: i think this one might be impossible, i think i did this on my own last time? XD

File: governance.md
Comment (bobcatfish) [inline governance.md:148]: i think we could delete the list below at this point b/c i think it's out of date and that script is probably the best source of info

**Review by bobcatfish — COMMENTED** [review]
Thanks @afrittoli ! Just some minor feedback from me :D

File: governance.md
Comment (vdemeester) [reply governance.md:73]: I think we should be, at least before we structure this more ? by "responsible", imho, this mainly means taking a decision if there is unresolved/unresolvable conflict on a TEP on wether it should go in or not

File: governance.md
Comment (afrittoli) [reply governance.md:17]: upz, of course, good catch

File: governance.md
Comment (afrittoli) [reply governance.md:24]: yes, I will add that

File: governance.md
Comment (afrittoli) [reply governance.md:73]: Added TEPs

File: governance.md
Comment (afrittoli) [reply governance.md:91]: indeed

File: governance.md
Comment (afrittoli) [reply governance.md:98]: done

File: governance.md
Comment (afrittoli) [reply governance.md:118]: Thanks, added a comment

**Review by vdemeester — COMMENTED** [review]
/lgtm

**Comment by afrittoli** [comment]
/hold
I would like to get an OK from all members of the governance team before this is merged.
/cc @ImJasonH @bobcatfish @abayer

File: governance.md
Comment (bobcatfish) [inline governance.md:21]: afaik andrew, jason and andrea are all elected until Feb 2022, just @vdemeester and i are Feb 2021

File: governance.md
Comment (bobcatfish) [inline governance.md:127]: extra o!

(or two?)

File: governance.md
Comment (bobcatfish) [inline governance.md:136]: on -> one

**Review by bobcatfish — COMMENTED** [review]
A couple nits from me (maybe the election dates bit is less of a nit and a bit more important) but anyway otherwise consider me approved! :D

File: governance.md
Comment (afrittoli) [reply governance.md:21]: Good catch, thank you!
I got it right for Andrew... and than copy pasted the wrong date 😅

File: governance.md
Comment (afrittoli) [reply governance.md:127]: fixed

**Comment by afrittoli** [comment]
> New changes are detected. LGTM label has been removed.

Addressed comments and more typos.

**Comment by bobcatfish** [comment]
/lgtm

thanks @afrittoli !

**Review by vdemeester — APPROVED** [review]
/hold cancel

**Pull request merged**
